resharper/test/data/CSharp/Intentions/QuickFixes/CachePropertyValue/LambdaTest.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace DefaultNamespace
+{
+    public class LabmdaTest
+    {
+        public void Test(Transform t)
+        {
+            var lambda = () => t.position + t.position + t.posi{caret}tion;
```

Comment:
Please add tests for void-returning lambda expressions e.g.
`Action action = () => DoSomething(t.position + t.position + t.posi{caret}tion);`

and lambdas under return statements e.g.
`return DoSomething(() =>  t.position + t.position + t.posi{caret}tion);`